pkg/kudoctl/packages/template/template_verify.go

```diff
@@ -0,0 +1,112 @@
+package template
+
+import (
+	"fmt"
+
+	engtask "github.com/kudobuilder/kudo/pkg/engine/task"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages/verifier"
+)
+
+// ParametersVerifier checks that all parameters used in templates are defined
+// checks that all defined parameters are used in templates
+type ParametersVerifier struct {
+}
+
+func (ParametersVerifier) Verify(pf *packages.Files) (warnings verifier.ParamWarnings, errors verifier.ParamErrors) {
+
```

Comment:
I think I am silly 🙈 but somehow empty lines in the beginning of methods bother me, sorry :D I think this is something that potentially we should unify on via linter...

Reply:
Done